aplib.net-demo/Assets/Scripts/Items/HealthPotion.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+
+namespace Assets.Scripts.Items
+{
+    /// <summary>
+    /// Health potion item that can be used to heal the player.
+    /// </summary>
+    [RequireComponent(typeof(PickupableItem))]
+    public class HealthPotion : Item
+    {
+        [SerializeField]
+        private int _healAmount = 50;
+
+        private HealthComponent _playerHealth;
+
+        protected override void Awake()
+        {
+            base.Awake();
+            _playerHealth = GameObject.FindGameObjectWithTag("Player").GetComponent<HealthComponent>();
+        }
+
+        /// <summary>
+        /// Uses the health potion to heal the player.
+        /// </summary>
+        public override void UseItem()
+        {
+            base.UseItem();
+            _playerHealth.ReduceHealth(-_healAmount);
```

Comment:
Dit vind ik een beetje gekke manier om een player extra health te geven. Zou je misschien een method `IncreaseHealth` toe kunnen voegen? Al is het maar een wrapper die dit doet, voor anderen zal dit beter aanvoelen denk.